graphql-dgs-client-example-apollo/src/test/java/subscription/integrationtest/SubscriptionIntegrationTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package subscription.integrationtest;
+
+
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.HostConfig;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Ports;
+import com.netflix.graphql.dgs.client.WebSocketGraphQLClient;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.reactive.socket.client.ReactorNettyWebSocketClient;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import reactor.core.publisher.Flux;
+import reactor.test.StepVerifier;
+
+import java.time.Duration;
+import java.util.Collections;
+
+
+@Testcontainers
+public class SubscriptionIntegrationTest {
+    private static final Logger logger = LoggerFactory.getLogger(SubscriptionIntegrationTest.class);
+
+    @Container
+    private static final GenericContainer<?> apolloContainer = new GenericContainer<>(
+            "amond/graphql-dgs-client-example-apollo"
```

Comment:
@amondnet I think this depends on the image that you are publishing to Dockerhub.
Although the test is interesting, and potentially worth to showcase as a pattern, I don't think it should reside in the framework.

Reply:
@berngp I removed that module.